recipe/meta.yaml

```diff
@@ -1,62 +1,77 @@
-{% set dependency_bigquery_storage = "google-cloud-bigquery-storage-core >=2.0.0,<3.0.0dev" %}
-{% set dependency_grpcio = "grpcio >=1.32.0,<2.0dev" %}
-{% set dependency_ipython = "ipython >=5.0.0" %}
-{% set dependency_pandas = "pandas >=0.23.0" %}
-{% set dependency_pyarrow = "pyarrow >=1.0.0,<3.0dev" %}
-{% set dependency_tqdm = "tqdm >=4.7.4,<=5.0.0dev" %}
 {% set name = "google-cloud-bigquery" %}
-{% set version = "2.4.0" %}
-{% set sha256 = "2d53d6fed1c25c9c0ce361dad1c64203c96b2344f7156357a501630d1cb08445" %}
+{% set version = "3.25.0" %}
+{% set build = 0 %}
+{% set sha256 = "5b2aff3205a854481117436836ae1403f11f2594e6810a98886afd57eda28509" %}
 {% set dev_url = "https://github.com/googleapis/python-bigquery" %}
-{% set doc_url = "https://googleapis.dev/python/bigquery/latest/index.html" %}
+{% set doc_url = "https://googleapis.dev/python/bigquery/latest" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  # Conda package for protobuf does not support Python 2.7 on Windows due to
-  # requirement on C++ 11. VS 2008 is not supported.
-  skip: true  # [win and vc<14]
+  number: {{ build }}
+  skip: true  # [py<38]
```

Comment:
This is the same as the other one where `python_requires=">=3.7"`.

Reply:
Similar answer, see line 43 about  importlib-metadata

Reply:
```suggestion
  # we skip py<38 following along conda-forge, to avoid importlib-metadata
  # which is needed for py37
  skip: true  # [py<38]
```